__tests__/getConfig.test.ts

```diff
@@ -1,121 +0,0 @@
-import getConfig from '../src/getConfig'
```

Comment:
I have to admit it hurts just a bit seeing all these tests get deleted, but it's for a good cause.